src/main/java/org/schemaspy/output/dot/schemaspy/DotSummaryFormatter.java

```diff
@@ -115,13 +112,4 @@ private boolean writeRelationships(Database db, Collection<Table> tables, boolea
         
         return wroteImplied;
     }
-
-    private static void markExcludedColumns(Map<Table, DotNode> nodes, Set<TableColumn> excludedColumns) {
-        for (TableColumn column : excludedColumns) {
-            DotNode node = nodes.get(column.getTable());
-            if (node != null) {
-                node.excludeColumn(column);
```

Comment:
Can this `DotNode` method be removed?

Reply:
Yes it can, only usage was in test.

Reply:
Okay, then please do so

Reply:
Since this comment and the second are related to the analysis and refactoring of DotNode I've created new issue for this #884